src/core/sys/windows/commctrl.d

```diff
@@ -3869,7 +3869,7 @@ static if (_WIN32_WINNT >= 0x501) {
     }
     alias LVINSERTGROUPSORTED* PLVINSERTGROUPSORTED;
 
-    alias int function(INT, INT, VOID*) PFNLVGROUPCOMPARE;
+    extern (Windows) alias int function(INT, INT, VOID*) PFNLVGROUPCOMPARE;
```

Comment:
Shouldn't it be `alias PFNLVGROUPCOMPARE = extern(Windows) int function(INT, INT, VOID*);` ?